src/main/kotlin/ch/bfh/habits/controllers/AuthController.kt

```
@@ -19,34 +19,53 @@
 import javax.servlet.http.HttpServletResponse
 
 @RestController
-class AuthController @Autowired constructor(
-        private val authenticationManager: AuthenticationManager,
-        private val tokenProvider: TokenProvider,
-        private val userService: UserService,
-    ) {
+class AuthController {
+
+    @Autowired
+    lateinit var authenticationManager: AuthenticationManager
```

detekt (1.23.7)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)